libs/s25main/nodeObjs/noSign.h

```diff
@@ -40,6 +40,10 @@ class noSign : public noDisappearingEnvObject
     /// An x,y zeichnen.
     void Draw(DrawPoint drawPt) override;
 
+    /// Return the amount of resources found, as drawn on the sign.
+    uint8_t GetSignAmount() const { return resource.getAmount(); }
```

Comment:
FYI: I combined those 2 functions into 1 which is more general: https://github.com/Return-To-The-Roots/s25client/pull/1373/files#diff-121534ca71eb140b4b21a437798817be9d09a6b8c8b1acdd824d465be1440ab0R44